usr/sbin/p11sak/p11sak.mk

```diff
@@ -11,12 +11,20 @@ usr_sbin_p11sak_p11sak_CFLAGS = -DLINUX -DPROGRAM_NAME=\"$(@)\"		\
 	-I${top_builddir}/usr/lib/config -I${top_builddir}/usr/lib/api	\
 	-I${srcdir}/usr/lib/api
 
+if !AIX
+usr_sbin_p11sak_p11sak_CFLAGS += -DLINUX
```

Comment:
I guess you should then remove the `-DLINUX` from line 8 (`usr_sbin_p11sak_p11sak_CFLAGS = -DLINUX ...`), otherwise -DLINUX is anyway there, even with AIX.